app/v1/controllers/DeleteReliefInvestmentsController.scala

```diff
@@ -71,4 +79,11 @@ class DeleteReliefInvestmentsController @Inject()(val authService: EnrolmentsAut
       case NotFoundError => NotFound(Json.toJson(errorWrapper))
     }
   }
+
+  private def auditSubmission(details: DeleteReliefInvestmentsAuditDetail)
+                             (implicit hc: HeaderCarrier,
+                              ec: ExecutionContext) = {
+    val event = AuditEvent("deleteReliefInvestments", "delete-relief-investments", details)
```

Comment:
The TXM document has these as "deleteReliefsInvestment" and "delete-reliefs-investment"